docs/docs-content/registries-and-packs/registries/oci-registry/oci-registry.md

```diff
@@ -27,10 +27,10 @@ The following type of OCI registries are supported:
 
 ## Limitations
 
-- Public OCI registries are not supported. You must use a private OCI registry that supports basic authentication.
+- Synchronization is not supported for Zarf registries.
```

Comment:
Just to confirm: Synchronization or automatic synchronization? :)

Reply:
Good catch @caroldelwing